### ISSUE TITLE
Wrong graphql document exports generated during codegen

### ISSUE THREAD

**Issue by dminikm** [jycouet/kitql#142]
## Description

I've encountered this issue when playing around with SvelteKit + graphql on a pre-existing (react + node) project which uses non-standard naming for graphql. All queries match JavaScript styled constant names, rather than the recommended graphql names.

For instance, `query AllContinents` becomes `query ALL_CONTINENTS`. This causes issues in KitQL.

## What's happening now
I've modified your demo example to show the issue: https://stackblitz.com/edit/jycouet-kitql-4ubezs.

KitQL generates this in `graphqlTypes.ts`:

```typescript
export const All_Continents = gql`
    query ALL_CONTINENTS {
  continents {
    ...ContinentFields
  }
}
    ${ContinentFields}`;
```

and

```typescript
export const All_ContinentsDocument = {"kind":"Document","definitions":[{"kind":"OperationDefinition","operation":"query","name":{"kind":"Name","value":"ALL_CONTINENTS"},"selectionSet":{"kind":"SelectionSet","selections":[{"kind":"Field","name":{"kind":"Name","value":"continents"},"selectionSet":{"kind":"SelectionSet","selections":[{"kind":"FragmentSpread","name":{"kind":"Name","value":"ContinentFields"}}]}}]}},...ContinentFieldsFragmentDoc.definitions]} as unknown as DocumentNode<All_ContinentsQuery, All_ContinentsQueryVariables>;
```

But `graphqlStores.ts` expects this:

```typescript
const res = await kitQLClient.request<Types.All_ContinentsQuery, Types.All_ContinentsQueryVariables>({
  skFetch: fetch,
  document: Types.AllContinentsDocument,
  variables, 
  operationName, 
  operationType, 
  browser
});
```
In short, the expected value is `AllContinentsDocument`, but the actual exported value is `All_ContinentsDocument`.

## What should happen
I would prefer if KitQL respected the chosen style, but that might be difficult to achieve.

- ALL_CONTINENTS => ALL_CONTINENTS_DOCUMENT
- AllContinents => AllContinentsDocument
- and so on ...

At the very least, it should generate the correct names.

**Comment by jycouet** [comment]
Not exactly what you requested, but it should build at least! :)
Let me know if it's good to you

**Comment by jycouet** [comment]
@all-contributors please add @dminikm for bug report

**Comment by dminikm** [comment]
@jycouet Thanks. Works just fine now.
